code/osapi/outwnd.cpp

```diff
@@ -221,9 +221,15 @@ void outwnd_print(const char *id, const char *tmp)
 		Assert( strlen(id)+1 < NAME_LENGTH );
 		outwnd_filter_struct new_filter;
 
+		new_filter.enabled = false;
+
 		strcpy_s(new_filter.name, id);
```

Comment:
I would put these on three consecutive lines, with whitespace afterward, because creating the struct is closely related to initializing it.

Reply:
@Goober5000 sounds good, made a change